.github/workflows/build_tests.yaml

```diff
@@ -50,18 +50,10 @@ jobs:
       run: |
         gcloud config set compute/zone us-east4-a
         gcloud config get compute/zone
-    - name: Install kjob
+    - name: Install xpk dependencies
       run: |
-        git clone https://github.com/kubernetes-sigs/kueue.git
-        cd kueue/cmd/experimental/kjobctl
-        make kubectl-kjob
-        mv ./bin/kubectl-kjob /usr/local/bin/kubectl-kjob
-    - name: Install kueuectl
-      run: curl -Lo ./kubectl-kueue https://github.com/kubernetes-sigs/kueue/releases/download/v0.8.1/kubectl-kueue-linux-amd64 && chmod +x ./kubectl-kueue && mv ./kubectl-kueue /usr/local/bin/kubectl-kueue
-    - name: Install xpk with pip and verify it executes corretly
-      run: |
-        pip install .
-        xpk --help
```

Comment:
Would you still retain this line `xpk --help` so that we can ensure that other steps have executed correctly?

Thank you!

Reply:
Sure thing, could you please give it a try in any environment you are using xpk(tests etc.) and will be using this makefile so we can be sure it is ok?